theme/static/style/style.css

```diff
@@ -450,4 +482,44 @@ pre code {
         padding-left: .8rem;
         text-align: left;
     }
+
+    /*features section*/
+    .features .w50 {
+        padding-top: 2rem;
+        min-width: 95%;
+        max-width: 95%;
+    }
+    .features .col-8 {
+        min-width: 100%;
+        max-width: 100%;
+    }
+    .features .col-8 .horizontally-stacked-list {
+        display: inline;
+    }
+    .features .col-8 li {
+        min-width: 100%;
+        max-width: 100%;
+    }
+    .features .col-3 {
+        min-width: 100%;
+        max-width: 100%;
+    }
+    .features img {
+        min-width: 95%;
+        max-width: 95%;
+        float: none;
+        margin: auto;
+        display: block;
+    }
+    .features li {
+        padding-left: 2.5%;
```

Comment:
IMO it is better to use 1.8rem as we do everywhere else. You can do the same for img as well, set max-width to 100% and update the border-size property to border-box to make sure the 100% contain all padding. margin and borders.